jr-stree/src/main/java/com/fasterxml/jackson/jr/stree/JacksonJrsTreeCodec.java

```diff
@@ -50,6 +59,9 @@ private JrsValue nodeFrom(JsonParser p) throws IOException
             return JrsBoolean.FALSE;
         case JsonTokenId.ID_NUMBER_INT:
         case JsonTokenId.ID_NUMBER_FLOAT:
+            if (_useBigDecimalForDouble) {
```

Comment:
Actually, this has bug -- now all integer numbers become `BigDecimal`s too. I'll fix it.